### DISCUSSION TITLE
bc-seedtool-cli on linux and windows

### DISCUSSION THREAD

**Discussion by gorazdko** [BlockchainCommons/Gordian-Developer-Community#13] [Show and tell]
Two PRs submitted:

* https://github.com/BlockchainCommons/bc-seedtool-cli/pull/28
* https://github.com/BlockchainCommons/bc-seedtool-cli/pull/29

Please help review and test!

![image](https://user-images.githubusercontent.com/25270775/93018858-4a334500-f5d3-11ea-8cfd-73cb04b852cf.png)

**Comment by wolfmcnally** [comment]
Merged all your dependency updates, looking forward to seeing the SSKR stuff updated so we can get the lastest version running everywhere!
